PRS4001-5000/PRS4381Gabriel.xml

```diff
@@ -45,11 +45,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPerson>
             <person sex="1">
```

Comment:
`sameAs="wd:Q4118033"
`
for persons, especially non-local, we always try to add wikidata ID